src/sentry/api/endpoints/organization_discover_saved_queries.py

```diff
@@ -108,6 +108,7 @@ def post(self, request, organization):
             organization=organization,
             name=data['name'],
             query=data['query'],
+            created_by=request.user,
```

Comment:
you may want to check if the `request.user.is_authenticated()` since it's possible to auth via tokens and i think the request.user would be an anonymous user